### ISSUE TITLE
Share over local network?

### ISSUE THREAD

**Issue by firebelly** [Fingertips/passengerpane#32]
There's an app called VirtualHostX, and it has an option to share hosts over the local network. So sites appear under the bonjour section in Safari bookmarks. Would it be possible to add that feature to the preference pane?

**Comment by alloy** [comment]
Yes, this feature will be added. Thanks for the pointer to VirtualHostX.

I can't promise when this will be added though…

**Comment by robertwbradford** [comment]
So at this point is there any way to access applications running on one machine from others in my LAN?

For example, I have 'my_app.local' running and available on 'machine_a'. I would like to access this from 'machine_b'. Before using the Passenger Pane I could access it at 'machine_a.local:3000' from machine_b. Is there a way to still do this?

**Comment by alloy** [comment]
Currently the only way with the current version of the pane is to add the host to the db's of the other machines…

Otherwise see [Kastner’s older fork](http://github.com/kastner/passengerpane) which uses bonjour. Something like this will eventually end up in our repo, but there are some hurdles to overcome. Eg, make it work automatically on different networks.

**Comment by martijn** [comment]
An mDNSResponder would be really cool, but I understand it's not easy. Users probably want control over the broadcasting of virtual hosts, too.
